QATCH/core/constants.py

```
@@ -564,19 +564,15 @@ def get_batch_param(batch, param=""):
         # print(f"batch_in_csv_file: {batch_in_csv_file}")
 
         # param does not exist, anywhere
-        if param_in_py_file == False and (
-            param_in_csv_file == False or batch_in_csv_file == False
-        ):
+        if param_in_py_file == False and (param_in_csv_file == False or batch_in_csv_file == False):
```

Ruff (0.15.10)

[error] Avoid equality comparisons to `False`; use `not param_in_py_file:` for false checks

Replace with `not param_in_py_file`

(E712)